checkpointctl.go

```diff
@@ -161,13 +172,23 @@ func inspect(cmd *cobra.Command, args []string) error {
 		)
 	}
 
-	if psTree {
+	if psTree || psTreeCmd {
```

Comment:
Same suggestion as the other PR, we can move the if block up and set `psTree = true`